microsoft-edge/webview2/release-notes.md

```diff
@@ -147,7 +147,149 @@ The following APIs have been promoted from Experimental to Stable in this Prerel
 ###### Runtime-only
 ###### SDK-only
 -->
-<!-- end of Jan 2024 Prerelease SDK -->
+<!-- end of Mar. 2024 Prerelease SDK -->
+
+
+<!-- ====================================================================== -->
+## 1.0.2415-prerelease
+
+Release Date: February 21, 2024<!-- todo: update date here & at top of file -->
+
+[NuGet package for WebView2 SDK 1.0.2415-prerelease](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.2415-prerelease)
+
+For full API compatibility, this Prerelease version of the WebView2 SDK requires the WebView2 Runtime that ships with Microsoft Edge version 122.0.2415.0 or higher.<!-- todo: fix version # -->
+
+
+<!-- ------------------------------ -->
+#### Breaking changes
+
+* Change in `LaunchingExternalURISchemeEventArgs` behavior.  If the `InitiatingOrigin` is an [opaque origin](https://html.spec.whatwg.org/multipage/browsers.html#concept-origin-opaque),the `InitiatingOrigin` reported in the event args will be its precursor origin.<!-- todo: link for more info -->
+
+* Change in the naming of `TextureStreamErrrorKind` enum. 

```

Comment:
Require more info about naming of enum.  The name changed from what to what?  So Dev must change the name likewise in their app code, correct?  Need to state that explicitly.
@victorhuangwq @peiche-jessica

Reply:
need to know & state the exact old name, for each of 3 platforms @victorhuangwq @peiche-jessica - what string will Dev search on to find the outdated name in their code?  for each platform.

is that a (breaking) change on Win32, or only for .NET & WinRT?
 
previous Win32 looks spelled ok:
https://learn.microsoft.com/en-us/microsoft-edge/webview2/reference/win32/webview2experimental-idl?view=webview2-1.0.2357-prerelease&preserve-view=true#corewebview2_texture_stream_error_kind
 
I'm skeptical about this whole breaking change.  enum was never spelled 'rrr': 
https://learn.microsoft.com/en-us/search/?scope=WebView2&view=webview2-winrt-1.0.2415-prerelease&terms=CoreWebView2TextureStreamErrrorKind - 0 hits

Reply:
rewrote, as renaming of enum's members, all 3 plats